frontend/packages/dev-console/src/components/topology/topology-types.ts

```diff
@@ -79,16 +79,31 @@ export interface TopologyDataObject<D = {}> {
   data: D;
 }
 
+export interface PodData {
```

Comment:
can we import this from frontend/packages/console-shared/src/types/pod.ts?

Reply:
It's already there in Pod.ts, I forgot to remove it from topology-types. fixed!!